### PR TITLE
LPS-117493 SF

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Hey @adolfopa, could you please review and complete the things that are missing?

1.[ Get the condition](https://github.com/liferay-lima/liferay-portal/pull/480/files#diff-0fc0f7561cfce04f47d0417e2f7b6948R48) when the user is searching everywhere. 
2. [Get the name](https://github.com/liferay-lima/liferay-portal/pull/480/files#diff-0fc0f7561cfce04f47d0417e2f7b6948R200) of the asset library or the site. Replace all the hardcoded ocurrences of "<small>Company Assets</small>"
3. Depending on if it belongs to an asset library or a site, the [icon](https://github.com/liferay-lima/liferay-portal/pull/480/files#diff-0fc0f7561cfce04f47d0417e2f7b6948R197) is different (icon "books" for asset libraries, icon "sites" for sites)

Thanks!

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/taglib/_item_selector.scss [hunk 1]
```diff
@@ -69,8 +69,8 @@
 }
 
 .item-selector {
-	.card-row {
-		align-items: center;
+	.card-row > .autofit-col-expand {
+		padding-top: 4px;
```
Comment (boton) [inline modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/taglib/_item_selector.scss:73]: I think is good idea use relative units maybe 0.25rem fit better :)

**Comment by adolfopa** [comment]
Just started reviewing :)

**Comment by adolfopa** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/482

**Pull request closed without merging**